03-MVC-AngularJS/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="myApp">
+    <head>
+        <meta charset="utf-8">
+        <title>Angular basics</title>
+        <meta name="description" content="Angular basics">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="css/normalize.css">
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular-route.js"></script>
+        <script type="text/javascript" src="js/script.js"></script>
+    </head>
+    <body>
+        <div ng-controller="mainController" class="wrapper">
+            <nav class= "wrapper--movieList">
+                <ul>
+                    <li ng-repeat="movie in movies.details" class="wrapper--item">
+                        <p>Title: {{ movie.title }}</p><br>
+                        <a href="#{{$index}}" ng-click="selectMovie(movie)">Show more details</a><br>
+                        <a href="" ng-click="deleteMovie($index)">Delete Movie</a>   |           
+                        <a href="" ng-click="editMovie($index)">Edit Movie</a>  
+                    </li>
+                    <li ng-hide="movies.details.length > 0" class="wrapper--item">Please add a new movie!</li>
+                </ul>
+            </nav>
+            <div class="wrapper--details">
+                <section ng-view ng-show="showDetails"></section>
+            </div>
+            <aside ng-controller="subController">
+                <form class="wrapper--form" ng-submit="submitAdd()">
```

Comment:
Try to use just one form to add, edit and also show details. Think this 3 forms as the same template with different controllers, it will make your life easier.